doc/Type/List.pod6

```diff
@@ -245,7 +245,7 @@ these invocations in a result list.
 
 Unlike C<map> it flattens non-itemized lists and arrays, so
 
-    say ((1, 2), <a b>).flatmap(&uc).join('|');     # 1|2|A|B
+    say ((1, 2), <a b>).flatmap(&uc).join('|');     # 1 2|A B
 
```

Comment:
While unrelated to the topic of this PR, it seems this example isn't the best one. The output shows no flattening.